handlers/reboot.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+pkill -f "main.py"
+git checkout master > ~/cleanr.log
+git pull origin master >> ~/cleanr.log
+python3 ~/Cleanr/main.py >> ~/cleanr.log 2>&1 &disown
```

Comment:
I'd rather have this relative to the project, so it works out of the box when cloning to any directory.

```suggestion
python3 ./main.py >> ~/cleanr.log 2>&1 &disown
```